web_client/src/types.ts

```diff
@@ -183,6 +183,7 @@ interface MIQAStore {
   currentFrameId: string | null;
   loadingFrame: boolean;
   errorLoadingFrame: boolean;
+  onDownloadProgress: (e: Event) => null | null;
```

Comment:
Should this be `(e: Event) => void` instead? (I am not a typescript expert so I'm not sure)

Reply:
Yes, you're right. I changed it in [9bb7713](https://github.com/OpenImaging/miqa/pull/657/commits/9bb7713fe1ad7c76d76d54829f5afb1b73072fdf)